spec/features/user_signs_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+feature "User signs in" do
+  background do
+    User.create(email: "john@domain.com", password: "password123", full_name: "john doe")
```

Comment:
You should use Fabricator for creating objects. This isn't bad but Fabricator has the advantage that it keeps all valid attributes for you. While if you'd add a new validation you would probably have to fix all these instead of 1 fabricator.